ga4gh/backend.py

```diff
@@ -37,8 +38,8 @@ def _parsePageToken(pageToken, numValues):
     return values
 
 
-def _getVariantSet(request, variantSetIdMap):
-    variantSetId = request.variantSetId
+def _getVariantSet(requestOrCompoundId, variantSetIdMap):
```

Comment:
I don't think this parameter overloading. This will get complicated quickly. Why not make a method ``_getVariantSetFromCompountId`` which parallels the ``_getDatasetFromCompoundId``?